mappings/net/minecraft/block/entity/ChestStateManager.mapping

```diff
@@ -0,0 +1,43 @@
+CLASS net/minecraft/class_5561 net/minecraft/block/entity/ChestStateManager
+	COMMENT Handles the viewer count for chest-like block entities.
+	FIELD field_27215 viewerCount I
+	METHOD method_31678 getViewerCount ()I
+	METHOD method_31679 isPlayerViewing (Lnet/minecraft/class_1657;)Z
+		COMMENT Determines whether the given player is currently viewing this chest.
+		ARG 1 entity
+	METHOD method_31680 getNumViewersInRange (Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;)I
```

Comment:
The `Num` is a bit awkward- maybe `getViewerCountInRange`? or even just `getViewersInRange` might be sufficient?